tangelo/toolboxes/molecular_computation/rdms.py

```diff
@@ -41,6 +48,96 @@ def matricize_2rdm(two_rdm, n_orbitals):
     return rho
 
 
+def compute_rdms(ferm_ham, exp_data, mapping, up_then_down):
```

Comment:
I am still thinking of how to support many possibilities to compute eigenvalues, with a nice interface. Another way of doing that in a more "modular" way would be to let the user provide a function to compute expectation value. We could let the user in charge of providing the right `exp_data` for the `get_exp_value_whatever...` function he/she would define. With that design, we would support direct evaluation, randomized CS, derandomized CS, ..., and so on.

Food for thought.

Reply:
Left for a separate PR, as per our discussion